code/controllers/subsystem/mobs.dm

```diff
@@ -7,7 +7,7 @@ SUBSYSTEM_DEF(mobs)
 	var/list/currentrun = list()
 	var/static/list/clients_by_zlevel[][]
 	var/static/list/cubemonkeys = list()
-	var/cubemonkeycap = 20
+	var/cubemonkeycap = 64
```

Comment:
how about just making it a damn config option

Reply:
@vuonojenmustaturska

Reply:
If the headmins want it, we can do it later